### PR TITLE
Schema Changes for Slider Control

### PR DESCRIPTION
**Pull request by srikant-ch5**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Review by matthoward366 — COMMENTED** [review]
@srikant-ch5 How are you handle the min/max values in the slider?

**Comment by srikant-ch5** [comment]
> @srikant-ch5 How are you handling the min/max values in the slider?

Hi @matthoward366 Please review PR for the min/max I have handled it in paramdef.

**Comment by srikant-ch5** [comment]
Hi @matthoward366 Please review this schema for slider with min and max.

File: common-pipeline/operators/uihints-v3-schema.json [hunk 1]
```diff
@@ -293,6 +294,16 @@
           "type": "number",
           "default": 1
         },
+        "min": {
```
[the remark this answers is not in the document]
Comment (matthoward366) [reply common-pipeline/operators/uihints-v3-schema.json:297]: Also, are you using increment?  We might need to update the comment.

[the remark this answers is not in the document]
Comment (srikant-ch5) [reply common-pipeline/operators/uihints-v3-schema.json]: Hi Matt, I have updated min_value and max_value and also updated increment description.

File: common-pipeline/operators/uihints-v3-schema.json [hunk 2]
```diff
@@ -283,13 +283,24 @@
...
             "timestampfield",
+            "slider",
             "code",
             "list",
             "hidden"
           ]
         },
         "increment": {
-          "description": "Determines the increment/decrement value for the spinner control only",
+          "description": "Determines the step value to increment/decrement for the slider control",
```
Comment (matthoward366) [inline common-pipeline/operators/uihints-v3-schema.json:293]: This should be spinner and slider control.

Comment (srikant-ch5) [reply common-pipeline/operators/uihints-v3-schema.json]: Sure. I have updated accordingly.

File: common-pipeline/operators/uihints-v3-schema.json [hunk 3]
```diff
@@ -283,13 +283,24 @@
...
             "timestampfield",
+            "slider",
             "code",
             "list",
             "hidden"
           ]
         },
         "increment": {
-          "description": "Determines the increment/decrement value for the spinner control only",
+          "description": "Determines the step value to increment/decrement for spinner and slider control",
```
Comment (matthoward366) [inline common-pipeline/operators/uihints-v3-schema.json:293]: Minor grammer update.  Should be `controls`

**Pull request merged**